files/extend.sh

```diff
@@ -59,27 +87,150 @@ commonName=supplied
 basicConstraints=critical,CA:TRUE
 keyUsage=keyCertSign,cRLSign
 subjectKeyIdentifier=hash
-authorityKeyIdentifier=issuer:always
+authorityKeyIdentifier=keyid:always
```

Comment:
Does changing this from `issuer` to `keyid` have any effect?

Reply:
Yeah, `keyid` causes the `Authority Key Identifier` field of the cert created by a CA to be populated from the `Subject Key Identifier` field of the CA cert --- which is the way it should be.

I think I misunderstood the OpenSSL manpage last time I worked on this script as `issuer` caused `Authority Key Identifier` to be populated with `DirName` and Serial` values that are definitely not what we want to be use as they aren't tied to the public key of the CA.